packages/beacon-node/src/chain/opPools/aggregatedAttestationPool.ts

```diff
@@ -428,7 +428,8 @@ export function isValidAttestationData(
   if (!ssz.phase0.Checkpoint.equals(data.source, justifiedCheckpoint)) return false;
 
   // Shuffling can't have changed if we're in the first few epochs
-  if (stateEpoch < 2) {
+  // Also we can't look back 2 epochs if target epoch is 1 or less
+  if (stateEpoch < 2 || targetEpoch <= 1) {
```

Comment:
The following issue https://github.com/ChainSafe/lodestar/issues/4547  is fixed with above change.